NuRadioReco/eventbrowser/apps/simulation_plots/sim_event_overview.py

```diff
@@ -35,8 +38,12 @@ def update_sim_event_3d(i_event, filename, station_id, juser_id):
     ariio = provider.get_arianna_io(user_id, filename)
```

Comment:
Is this ARIANNA related or is this a very unfortunate remainder? 
Can also be cleaned up later, if this is harder to fix.